content/learn/getting-started/node-javascript-language.pt.md

```diff
@@ -0,0 +1,39 @@
+---
+title: how-much-javascript-do-you-need-to-know-to-use-nodejs
+displayTitle: 'Quanto de JavaScript precisas de saber para usar a Node.js?'
+description: 'Se estiveres apenas começando com a JavaScript, quão profundamente precisas de saber a linguagem?'
```

Comment:
I am using the basic and standard Portuguese grammar, I will not change it because the pronoun used is *Tu* and the form of verbs in this pronoun is always in the plural, for example: "Tu falas", "Tu dizes", "Tu fazes", "Tu estás", "Se Tu estiveres" etc. I am not translating based on one specific variation of Portuguese, I am just using the rules present or must be present in all variations.

I am not using exceptions like "Você" which is made a rule on the Brazillian form of speaking.

I do not see the necessity of changing it, and since there are no grammar or syntactical errors, it should remain as it is.